sklearn/inspection/_partial_dependence.py

```diff
@@ -418,4 +497,21 @@ def partial_dependence(estimator, X, features, *, response_method='auto',
     averaged_predictions = averaged_predictions.reshape(
         -1, *[val.shape[0] for val in values])
 
-    return averaged_predictions, values
+    if kind == 'legacy':
+        warnings.warn(
+            "A Bunch will be returned in place of 'predictions' from 0.26 with"
```

Comment:
Use the word "version" please

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/e45c480cb8cc14834aa4b19c71379a2f67883efd